fakeData.py

```diff
@@ -0,0 +1,135 @@
+import numpy as np
+import random
+from faker import Faker
+import pandas as pd
+
+"""
+
+Copmpile a fake dataset resembling the dataset Michael collected on his trip to Guatemala in.
+
+The fake dataset uses relative proportions of the tree types in the original dataset as 
+probabilities for selection of the tree types of this dataset. So, although it is not the original data--
+and the names are fake!--it resembles the patterns of the co-op.
+
+"""
+
+fake = Faker('es_MX') # faker does not have options for any American Spanish varieties besides Mexico,
+# so we select Mexico because (1) you can find coffee plantations in Mexico, and (2) it is just north of Guatemala
+
+numFarmers = 40 # how many farms/farmers we want in our fake dataset
+
+
+# list the species of trees plated on the co-op
+treeNames = ['borbon', 'catuai', 'e14', 'catura']
+
+# utilize probabilities of original dataset
+freqProb = {'borbon': 0.5534, 'catuai': 0.4191, 'e14': 0.0046, 'catura': 0.02299} 
+
+# even if order changes, the probabilities still correspond to the strings
+#lsTreeTypes = random.choices()
+
+
+lsOfFarmers = [] # create a list to append these farmers to
+lsOfTrees = [] # create a list to hold lists for each farmer's trees
+lsOfCuerdas = [] # creat a list to hold the number of cuerdas
+lsOfAges = []
```

Comment:
inline comments can be omitted entirely when names of variables are unambiguous, which these are.
I think `FarmerList`, `TreeList` is a better pattern for readabillity, but I do like how they all start the same substring.

Reply:
```
In [22]: def fakeData(numFarms, numFarmers): 
    ...:     freqProb = {'borbon': 0.5534, 'catuai': 0.4191, 'e14': 0.0046, 'catura': 0.02299}  
    ...:     trees = random.choices(list(freqProb.keys()), weights=list(freqProb.values()), k =numFarms) 
    ...:     farmers = ['a', 'b', 'c'] # create fake name list of length numFarmers 
    ...:     randfarmers = random.choices(farmers, k = numFarms) 
    ...:     return (randfarmers, trees) 
```

Reply:
add in random year as a third column.